src/api/config/routes.rb

```diff
@@ -769,6 +769,7 @@ def self.public_or_about_path?(request)
       resource :staged_requests, controller: 'staging/staged_requests', only: [:create, :destroy], constraints: cons
     end
 
+    resources :excluded_requests, controller: 'staging/excluded_requests', only: [:index], constraints: cons
     resource :excluded_requests, controller: 'staging/excluded_requests', only: [:create, :destroy], constraints: cons
```

Comment:
BTW, (unrelated to this PR) why is this route in singular?

Reply:
> BTW, (unrelated to this PR) why is this route in singular?

For the `:destroy` route the id was not required (multiple excluded requests can be unexcluded at a time). And as the `:create` route doesn't need the id, it was decided to merge both routes in the same `resource` helper.

Now that the `:index` route is added, yes, you are right: `:index` and `:create` routes can be defined in the `resources` helper, and the `:destroy` route can be defined in the `resource` helper. I can do this in another pull request, after #7161 gets merged (the refactor of staging routes is done).